resources/cygnus-translators/README.md

```diff
@@ -1,7 +1,14 @@
-# Cygnus translators
+#<a name="top"></a>Cygnus translators
 
-## `cygnus-translator-0.1-to-0.2.sh`
+Content:
 
+* [`cygnus_translator_0.1_to_0.2.sh`](#section1)
+* [`cygnus_translator_0.1_to_0.3.sh`](#section2)
+* [`cygnus_translator_0.2_to_0.3.sh`](#section3)
+* [`cygnus_translator_0.9.0_to_0.10.0_hdfs.sh`](#section4)
```

Comment:
May this name lead to think that if I'm using for instance 0.5.0 (i.e. I'm not using exactly the version name mentioned in the script name) and I want to go to 0.10.0 I do *not* need to follow the procedure?

Maybe the formulation used in Orion could help (https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/admin/install.md#migrating-the-data-stored-in-db): "Upgrading to 0.21.0 and beyond from a pre-0.21.0 version" although not sure how to apply it to a .sh script name.